donate/templates/pages/base_page.html

```diff
@@ -12,49 +12,71 @@
     <meta property="og:title" content="{{ page.title }}" />
 {% endblock meta_tags %}
 
-
 {% block body_top %}
 {% endblock body_top %}
 
+{% block navigation %}
+    <div class="header__menus header__menus--mobile" data-mobile-menu>
+        {% primarynav %}
+    </div>
+{% endblock navigation %}
 
 {% block header %}
     <header class="header">
-        <div class="header__inner wrapper wrapper--large">
-            <a href="/">
-                <img class="header__logo" src="/_images/mozilla.svg" alt="Logo for donate">
-            </a>
+        <div class="header__container">
 
             <div class="header__menu-toggle">
-                <span>Menu</span>
-                <button class="button button-menu-toggle" data-mobile-menu-toggle aria-haspopup="true" aria-label="Mobile menu toggle">
-                    <span class="button-menu-toggle__line"></span>
-                    <span class="button-menu-toggle__line"></span>
-                    <span class="button-menu-toggle__line"></span>
-                    <span class="button-menu-toggle__line"></span>
+                <span class="header__menu-label">Menu</span>
+                <button class="burger button-menu-toggle" data-mobile-menu-toggle  aria-haspopup="true" aria-label="Mobile menu toggle">
+                    <div class="burger__bar burger__bar--top"></div>
+                    <div class="burger__bar burger__bar--middle"></div>
+                    <div class="burger__bar burger__bar--bottom"></div>
                 </button>
             </div>
 
-            <section class="header__menus header__menus--desktop">
-                {% primarynav %}
-            </section>
+            <a class="header__logo-link" href="/">
+                <img class="header__logo" src="/_images/mozilla-logo-white.svg" alt="Mozilla">
+            </a>
 
-            <section class="header__menus header__menus--mobile" data-mobile-menu>
-                {% primarynav %}
-            </section>
         </div>
     </header>
 {% endblock header %}
 
 {% block footer %}
-    <footer class="footer">
-        <div class="footer__inner wrapper wrapper--large">
-            <div class="footer__nav-container">
-
+<div class="footer">
+    <div class="footer__container">
+        <div class="footer__nav footer-links">
+            <ul class="footer-links__container">
+                <li class="footer-links__item">
+                    <a class="link link--dark-bg" href="https://twitter.com/mozilla" target="_blank">Twitter</a>
```

Comment:
worth adding `rel="noopener"` to these links that open in a new tab - https://developers.google.com/web/tools/lighthouse/audits/noopener

Reply:
Done :)